### PR TITLE
add FastFusionNet results

### PR DESCRIPTION
**Pull request by felixgwu**

### CODE REVIEW

File: SQuAD/inference/dawn_fastfusionnet-1080ti.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+    "version": "v1.0",
+    "author": "Wu et al. (Cornell, SayMosaic, Google)",
+    "authorEmail": "dawnbench@cornell.edu",
+    "framework": "Pytorch v0.3.1",
+    "model": "FastFusionNet",
+    "hardware": "1 NVidia GTX-1080 Ti",
+    "timestamp": "2019-02-27",
+    "codeURL": "https://github.com/felixgwu/FastFusionNet",
```
Comment (codyaustun) [inline SQuAD/inference/dawn_fastfusionnet-1080ti.json:9]: Please specify the commit hash either in the `codeURL` or as a separate `commitHash` field

File: SQuAD/train/dawn_fastfusionnet-1080ti.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "v1.0",
+    "author": "Wu et al. (Cornell, SayMosaic, Google)",
+    "authorEmail": "dawnbench@cornell.edu",
+    "framework": "Pytorch v0.3.1",
+    "model": "FastFusionNet",
+    "hardware": "1 NVidia GTX-1080 Ti",
+    "timestamp": "2019-03-01",
+    "codeURL": "https://github.com/felixgwu/FastFusionNet",
```
Comment (codyaustun) [inline SQuAD/train/dawn_fastfusionnet-1080ti.json:9]: Same as above. Please specify the commit hash either in the `codeURL` or as a separate `commitHash` field

File: SQuAD/inference/dawn_fastfusionnet-1080ti.json (hunk 1 above)
Comment (codyaustun) [reply SQuAD/inference/dawn_fastfusionnet-1080ti.json:9]: Also looking at the repo, I see that you have instructions for training. Can you also include instructions for inference?

**Review by codyaustun — CHANGES_REQUESTED** [review]
Looks good overall. I requested some minor changes.

File: SQuAD/train/dawn_fastfusionnet-1080ti.json (hunk 2 above)
Comment (felixgwu) [reply SQuAD/train/dawn_fastfusionnet-1080ti.json:9]: Done.

File: SQuAD/inference/dawn_fastfusionnet-1080ti.json (hunk 1 above)
Comment (felixgwu) [reply SQuAD/inference/dawn_fastfusionnet-1080ti.json:9]: I've added commitHash and the instructions for inference. Thanks!

**Comment by felixgwu** [comment]
Hi @codyaustun, the commitHash and the inference instructions have been added.
Is there anything else you want to add?

**Comment by codyaustun** [comment]
Looks good to me!

**Review by codyaustun — APPROVED** [review]

**Comment by felixgwu** [comment]
Thanks!

**Comment by felixgwu** [comment]
Hi @codyaustun 
It looks like there is a bug when processing the tsv file.
It uses the training time after the first epoch instead of the the time spent after the 4th epoch (when the model gets F1 75).

image

**Comment by codyaustun** [comment]
@felixgwu good catch! The f1Score column should be [0, 1] not [0, 100]. Can you please fix and submit a pull request?

**Comment by felixgwu** [comment]
@codyaustun 
Since I've deleted my old fork, I create a new pull request here instead: https://github.com/stanford-futuredata/dawn-bench-entries/pull/81

**Pull request merged**